### PR TITLE
Adding netcoreapp1.2corert configuration to System.Collections

### PR DESCRIPTION
**Pull request by SedarG**
@jkotas, PTAL

/cc @dotnet/corert-contrib

### CODE REVIEW

File: src/System.Collections/src/System.Collections.csproj [hunk 1]
```diff
@@ -34,6 +34,14 @@
       <OSGroup>$(InputOSGroup)</OSGroup>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp1.2corert'">
+    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
+      <TargetGroup>netcoreapp1.2corert</TargetGroup>
+    </ProjectReference>
+    <ProjectReference Include="..\..\System.Resources.ResourceManager\src\System.Resources.ResourceManager.csproj">
```
Comment (jkotas) [inline src/System.Collections/src/System.Collections.csproj]: You should not need the ResourceManager reference for CoreRT

Comment (SedarG) [inline src/System.Collections/src/System.Collections.csproj]: Good catch, thanks! I'm updating the csproj.

**Comment by weshaggard** [comment]
cc @joperezr configuration change to what with the dev/eng merge.

**Comment by karelz** [comment]
@weshaggard @joperezr do you want to merge it or wait for post dev/api RI?

**Comment by weshaggard** [comment]
If it is ready go ahead and merge it. We can handle the configuration merge.

FYI for @jkotas and @SedarG I'm disabling commit access to corefx master tomorrow to get the dev/eng branch merged into master.

**Comment by SedarG** [comment]
@dotnet-bot please test Innerloop OSX Release Build and Test

**Pull request merged**